### ISSUE TITLE
Kodi is not able to show station name from online radio streams

### ISSUE THREAD

**Issue by Mannshoch** [xbmc/xbmc#18332]
### Describe the bug

 If I send Kodi to play these stream http://stream.srg-ssr.ch/drs3/mp3_128.m3u
I'm able to get all need Date with help from  https://kodi.wiki/view/InfoLabels

The only Data I could not get is the Station name. This Information have to be available because VLC media player is able to show me the name.

As far I get from VLC this kind of Metadata came from tag called icy-name and it seems there are several icy-{TAG}

(See my post on: https://forum.kodi.tv/showthread.php?tid=356236)

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

The Station name appear if you play a radiostream

## Actual Behavior

the name of the .m3u appear.

## Possible Fix

https://github.com/the-black-eagle/script.streaming.radio.helper/blob/master/resources/lib/utils.py#L330-L377

### To Reproduce

1. Open http://stream.srg-ssr.ch/drs3/mp3_128.m3u
2. check the Information that appear

### Debuglog

The debuglog can be found here:
```
2020-08-26 06:00:00.004 T:3011652064  NOTICE: VideoPlayer::OpenFile: /storage/radiozurisee.m3u
2020-08-26 06:00:00.366 T:2871812976  NOTICE: Creating InputStream
2020-08-26 06:00:02.720 T:2871812976  NOTICE: Creating Demuxer
2020-08-26 06:00:02.740 T:2871812976  NOTICE: Opening stream: 0 source: 256
2020-08-26 06:00:02.741 T:2871812976  NOTICE: Finding audio codec for: 86017
2020-08-26 06:00:02.741 T:2871812976  NOTICE: CDVDAudioCodecFFmpeg::Open() Successful opened audio decoder mp3float
2020-08-26 06:00:02.741 T:2871812976  NOTICE: Creating audio thread
2020-08-26 06:00:02.742 T:2362999664  NOTICE: running thread: CVideoPlayerAudio::Process()
2020-08-26 06:00:02.753 T:2362999664  NOTICE: Creating audio stream (codec id: 86017, channels: 2, sample rate: 44100, no pass-through)
2020-08-26 06:00:03.120 T:2871812976 WARNING: CDVDMessageQueue(video)::Put MSGQ_NOT_INITIALIZED

```

### Your Environment
Used Operating system:

 - [x] Raspberry-Pi

 - Operating system version/name:
 - Kodi version: Kodi (18.6 Git:leia_pi4_18.6-Leia)

**Comment by DaVukovic** [comment]
By reading the answer at the linked thread from @the-black-eagle this sounds more like a feature request rather than a bug report. 

Maybe @the-black-eagle can confirm?

**Comment by the-black-eagle** [comment]
I wrote my radio addon specifically because this info was missing in Kodi core although having taken a very quick glance at the code, it looks like maybe it should be available.
https://github.com/xbmc/xbmc/blob/293e0fc15f24a1edc12a8e754868fa47df37fdda/xbmc/filesystem/ShoutcastFile.cpp#L69-L71
I added a bit of logging to see if that is indeed returning a station name and it is. However, that appears to be over-written by the artist and track information as that is what is contained in the Title label.  I've not yet looked to see where that might happen but I guess as Kodi does parse out the station name from the header then this can be considered a bug.
